README.md

```diff
@@ -25,23 +32,22 @@ A thin wrapper around the
 ## Supported event keys
 
 - `:breadcrumbs` - a collection of `Breadcrumb` maps. See below.
-- `:checksum-for` - a `String` whose checksum should be associated with the event.
-- `:checksum` - a `String` which _is_ the checksum for the event, if you're doing your own checksumming.
-- `:culprit` - a `String` or a `StackTraceElement` which identifies the so-called culprit. This is not very well documented at Sentry, but [here is an example](https://docs.sentry.io/clients/ruby/integrations/puma/) of how they use the `String` variant in a Ruby application.
+- `:dist` - a `String` which identifies the distribution.
 - `:environment` - a `String` which identifies the environment.
-- `:event-id` - a `UUID` to identify the event. Autogenerated if not specified.
+- `:event-id` - a `String` id to use for the event. If not provided, one will be automatically generated.
 - `:extra` - a map with `Keyword` or `String` keys (or anything for which `clojure.core/name` can be invoked) and values which can be JSON-ified. If `:throwable` is given, this will automatically include its `ex-data`.
 - `:fingerprint` - a sequence of `String`s that Sentry should use as a [fingerprint](https://docs.sentry.io/learn/rollups/#customize-grouping-with-fingerprints).
-- `:interfaces` - a map with `Keyword` or `String` keys and values that can be JSON-ified. See the [Sentry documentation on interfaces](https://docs.sentry.io/clientdev/interfaces/) for information on the expected format.
 - `:level` - a `Keyword`. One of `:debug`, `:info`, `:warning`, `:error`, `:fatal`. Probably most useful in conjunction with `:message` if you need to report an exceptional condition that's not an exception.
 - `:logger` - a `String` which identifies the logger.
 - `:message` - a `String`. This is a different message than that of a Throwable.
```

Comment:
We changed what `message` looks like:

https://github.com/getsentry/sentry-java/blob/c322651caaf2c1d5d1a39c485cc5b940e952cfbc/sentry/src/main/java/io/sentry/SentryEvent.java#L75-L81

Reply:
Yup that is true, but in terms of the API to sentry-clj, it accepts a string, which then constructs a Message (object) with the String as the `setMessage`. I do see however that it does now accept a String list of Params. Happy to add that in too :-)

Reply:
That's OK. It does limit then the clojure users because in terms of grouping, if you're able to capture the "raw" string (uninterpolated) Sentry can group on that.

message: `username %s logged in`
formatted: `username dharrigan logged in`
params: `["dharrigan"]`.

Sentry now can show the user `formatted` but uses `message` to group.

Reply:
On it :-)

Reply:
Done.